sdk/lib/dmilib/precomp.h

```diff
@@ -0,0 +1,9 @@
+#ifndef _DMILIB_PCH_
+#define _DMILIB_PCH_
+
+#include <ntifs.h>
```

Comment:
See if either `<wdm.h>` (preferred) or `<ntddk.h>` works. They provide smaller subsets of the kernel headers so should be preferred unless something from the larger header is needed.